client/jetbrains/src/main/java/com/sourcegraph/cody/CodyToolWindowContent.java

```diff
@@ -126,6 +127,16 @@ public CodyToolWindowContent(@NotNull Project project) {
     addWelcomeMessage();
   }
 
+  @NotNull
+  private static JButton createWideButton(String text) {
```

Comment:
I've tried to explicitly marked null safety in all places in the existing code, and I suggest keeping this.

```suggestion
  private static JButton createWideButton(@NotNull String text) {
```

If this is an unneeded overhead that I'm not seeing, let me know, and we can discuss!